xml/project.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "sample-xml",
+    "id": "sample-xml",
+    "projectType": "custom",
+    "sourceLocale": "en-US",
+    "pseudoLocale": "zxx-XX",
+    "plugins": [
+        "ilib-loctool-xml"
+    ],
+    "excludes": [
+        "xliffs",
+        "locale",
+        "schemas"
+    ],
+    "settings": {
+        "xliffsDir": "./xliffs",
+        "locales":[
+            "de-DE",
+            "nl-NL"
+        ],
+        "xml": {
+            "schemas": [
+                "./schemas"
+            ],
+            "mappings": {
+                "xml/res/**/*.xml": {
+                    "schema": "android-resources-schema",
```

Comment:
Question)
looks like `android-resources-schema` file is missing. is it intended?
and we have plugin for android localization. Could this plugin replace them?

Reply:
It comes with the xml plugin as the default schema:

https://github.com/iLib-js/ilib-loctool-xml/blob/main/schemas/android-resource-schema.json

The xml plugin could eventually replace the android plugins, yes.